From oliver@neukum.org  Thu Feb 12 11:42:54 2009
From: Oliver Neukum <oliver@neukum.org>
Date: Mon, 9 Feb 2009 10:03:49 +0100
Subject: USB: serial: remove recourse to generic method
To: Greg KH <greg@kroah.com>
Message-ID: <200902091003.50794.oliver@neukum.org>
Content-Disposition: inline


This removes the fallback to the generic method. It is cleaner to
explicitely request it. Introducing this was my mistake. This will
be solved by an explicit test and the driver being allowed to request
what it needs to be done upon resumption.

Signed-off-by: Oliver Neukum <oneukum@suse.de>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/serial/usb-serial.c |    1 -
 1 file changed, 1 deletion(-)

--- a/drivers/usb/serial/usb-serial.c
+++ b/drivers/usb/serial/usb-serial.c
@@ -1229,7 +1229,6 @@ static void fixup_generic(struct usb_ser
 	set_to_generic_if_null(device, read_bulk_callback);
 	set_to_generic_if_null(device, write_bulk_callback);
 	set_to_generic_if_null(device, shutdown);
-	set_to_generic_if_null(device, resume);
 }
 
 int usb_serial_register(struct usb_serial_driver *driver)
